README.md

```diff
@@ -146,7 +146,7 @@ Using the Library Interfaces
 We provide a simple C-style interface to use Mt-KaHyPar as a library.  The library can be built and installed via
 
 ```sh
-make install.library
+sudo make install.mtkahypar
```

Comment:
why install the library system-wide? rather create a library file that people can link with

Reply:
i did not found any solution to this without copying everything